js/hoot/model/rest.js

```diff
@@ -585,6 +585,27 @@ Hoot.model.REST = function (command, data, callback, option) {
             if (error) {
                 return callback(_alertError(error, 'Get Translations failed! For detailed log goto Manage->Log'));
             }
+
+            // Split DEFAULT and not to organize DEFAULT on top and alpha for both
+            try {
+                var dTrans = _.groupBy(resp,'DEFAULT');
+                dTrans.true.sort(function(x,y){return d3.ascending(x.NAME.toLowerCase(),y.NAME.toLowerCase());});
+
+                if(dTrans.false && dTrans.undefined){ dTrans.undefined.concat(dTrans.false); }
+                else if(dTrans.false && !dTrans.undefined){ dTrans.undefined = dTrans.false; }
+
+                if(dTrans.undefined){
+                    dTrans.undefined.sort(function(x,y){return d3.ascending(x.NAME.toLowerCase(),y.NAME.toLowerCase());});
+                    resp = dTrans.true.concat(dTrans.undefined);
+                } else {
+                    resp = dTrans.true;
+                }
+
+
+            } catch (eTrans) {
+                resp.sort(function(x,y){ return (x.DEFAULT === true) ? 0 : x.DEFAULT ? -1 : 1; });
+            }
+
```

Comment:
Seems to work, but this code is hella confusing.

The purpose is to sort by DEFAULT = true, then false, then undefined and alphabetically within those?
Could we just use a sort function that utilized both rules?

I'll try to create a proposed sort function and mocha test for this.